.snyk

```diff
@@ -2,5 +2,5 @@ version: v1.5.0
 ignore: {}
 patch:
   'npm:request:20160119':
-    - splunk-bunyan-logger > splunk-logging > request:
-        patched: '2016-11-04T14:40:50.085Z'
+    - bunyan-logger > splunk-bunyan-logger > splunk-logging > request:
```

Comment:
If bunyan-logger has been patched (https://github.com/nhsuk/bunyan-logger/pull/2) does this still need doing in connecting-to-services.